test/e2e/common/configmap.go

```diff
@@ -78,9 +78,9 @@ var _ = Describe("[sig-api-machinery] ConfigMap", func() {
 	})
 
 	/*
-		    Testname: configmap-envfrom-field
-		    Description: Make sure config map value can be used as an source for
-			environment variables in the container (on container.envFrom field)
+		Release: v1.9
+		Testname: ConfigMap from environment variables
+		Description: Create a Pod with a environment source from configMap. All config map values MUST be available as environment variables in the container.
```

Comment:
Should this description also note that the test is verifying that the environment variable names can be given a prefix?

Reply:
Actually it is a mapping right?